sympy/parsing/mathematica.py

```diff
@@ -8,126 +9,218 @@ def mathematica(s):
     return sympify(parse(s))
 
 
-class _Parse(object):
-    '''An instance of this class converts basic Mathematica expression to
-    Python.'''
+class parse(object):
+    '''parse class converts basic Mathematica expression to Python.'''
 
-    replaces1 = (
-        (' ', ''),
-        ('^', '**'),
-    )
+    # translate
+    F = {
+        # (func in Mathematica, number of arguments): func in sympy
+        ('Mod', 2): 'Mod',
+        ('Sqrt', 1): 'sqrt',
+        ('Exp', 1): 'exp',
+        ('Log', 1): 'log',
```

Comment:
Log may have one or two parameters.

Reply:
In F_swap dictionary, ('Log', 2): 'log' is defined.